imagepipeline/src/main/java/com/facebook/imagepipeline/producers/ResizeAndRotateProducer.java

```diff
@@ -164,6 +167,22 @@ private void doTransform(EncodedImage encodedImage, boolean isLast) {
           ret.setImageFormat(ImageFormat.JPEG);
           try {
             ret.parseMetaData();
+            int fwidth=ret.getWidth();
```

Comment:
Nit: coding style, please use 2 spaces instead of tabs, and add spaces around operators and keywords.
This is just a convention we follow in this repo.